src/app/fyle/my-expenses/my-expenses.page.ts

```
@@ -1759,4 +1781,95 @@ export class MyExpensesPage implements OnInit {
+    expenses: PlatformExpense[]
+  ): Observable<PlatformExpense[]> {
+    let updatedExpensesList = expenses;
+    // Create a stop signal that emits after 30 seconds
+    const stopPolling$ = timer(30000);
+    return timer(5000, 5000).pipe(
+      exhaustMap(() => {
+        const params: ExpensesQueryParams = { queryParams: { id: `in.(${dEincompleteExpenseIds.join(',')})` } };
+        return this.expenseService.getExpenses({ ...params.queryParams }).pipe(
+          map((updatedExpenses) => {
+            updatedExpensesList = this.updateExpensesList(updatedExpensesList, updatedExpenses, dEincompleteExpenseIds);
+            dEincompleteExpenseIds = this.filterDEIncompleteExpenses(updatedExpenses);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)